Formula/aws-vault.rb

```diff
@@ -0,0 +1,23 @@
+class AwsVault < Formula
+  desc "Securely store and access AWS credentials in development environments"
+  homepage "https://github.com/99designs/aws-vault"
+  url "https://github.com/99designs/aws-vault/archive/v4.6.0.tar.gz"
+  sha256 "058f20dcfafad2641c35b6970909d0dedd71db66450d820f8f1fb457abcf50a0"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOOS"] = "linux"
+    ENV["GOARCH"] = "amd64"
+
+    system "make", "build"
+    bin.install "aws-vault"
+
+    zsh_completion.install "completions/zsh/_aws-vault"
+    bash_completion.install "completions/bash/aws-vault"
+  end
+
+  test do
+    assert_match("aws-vault: error: required argument 'profile' not provided, try --help", shell_output("#{bin}/aws-vault login 2>&1", 1))
```

Comment:
I'm wondering if this is too simple a test, but this at least checks some logic (the presence of a `profile` argument).

Reply:
This is OK as the first step. Even though this is a simple check, it will check the consistency of output between versions.